Public/deploy/robots.txt

```diff
@@ -1,6 +1,6 @@
 # All robots allowed
 User-agent: *
-Disallow:
+Disallow: /builds/*
```

Comment:
It must have been pure coincidence that the search engines finished their index as we made this change, because it didn't go live 😂

Reply:
It didn't? But I saw the diff?

Reply:
d8ea75724e261f091a416273ef57f25e646de887

Reply:
Oooh, it's not in `deploy`

Reply:
Yep 😬

Reply:
I checked it on the live site too, it's not there.